CONTRIBUTING.md

```diff
@@ -13,13 +13,37 @@ pnpm run setup
 
 Note for Windows: Use the latest version of [Git Bash](https://gitforwindows.org/)
 
+### General prerequisites
+
+1. Install [`pnpm`](https://pnpm.js.org/) (for installing npm dependencies)
+1. Install [`docker`](https://www.docker.com/products/docker-desktop) (for managing test databases)
+
 ### Developing Prisma Client JS
 
-2. `cd src/packages/client`
-3. `ts-node fixtures/generate.ts ./fixtures/blog/ --skip-transpile`
-4. `cd fixtures/blog`
-5. `prisma migrate save --name init --experimental && prisma migrate up --experimental`
-6. `ts-node main.ts`
+1. `cd src/packages/client`
+1. `ts-node fixtures/generate.ts ./fixtures/blog/ --skip-transpile`
+1. `cd fixtures/blog`
+1. `export DB_URL=YOUR_POSTGRES_DB_URL`  
+   For this step you might find our [docker-compose setup](./src/docker) helpful
+1. `npx prisma migrate save --name init --experimental && prisma migrate up --experimental`
+1. `ts-node main`
+
+### Running integration tests for Prisma Client JS
+
+Start the test databases (see [readme](./src/docker) for various ways to run these)
+
+1. `cd src/docker`
+1. `docker-compose up -d`
+
+Start the tests
+
+1. `cd src/packages/cli`
+2. `pnpm run test:postgresql && pnpm run test:mysql && pnpm run test:mariadb`
```

Comment:
```suggestion
2. `pnpm run test`
```

You missed sqlite here and stuff, all are inside the generic "test"

Reply:
nice